src/monmove.cpp

```diff
@@ -62,6 +62,8 @@ static const efftype_id effect_harnessed( "harnessed" );
 
 static const species_id ZOMBIE( "ZOMBIE" );
 
+static const std::string flag_AUTODOC_COUCH( "AUTODOC_COUCH" );
```

Comment:
Out of curiosity: why is some code using a string literal (e.g. in `mattack::nurse_operate`) and some code a global constant?

Reply:
I used the constant when the string was used more than once, is that wrong?

Reply:
It's definitively not wrong. It's just inconsistent and makes people wonder if there's a meaning to it. Especially as the name of the constant is the same as the string literal. Reading `flag_AUTODOC_COUCH` doesn't convey more information than "AUTODOC_COUCH".

Reply:
So I should use the constant everywhere?